Rubberduck.Core/UI/CodeExplorer/Commands/RemoveCommand.cs

```diff
@@ -2,24 +2,36 @@
 using Rubberduck.Interaction;
 using Rubberduck.Navigation.CodeExplorer;
 using Rubberduck.Resources.CodeExplorer;
+using Rubberduck.UI.Command;
+using Rubberduck.VBEditor;
 using Rubberduck.VBEditor.ComManagement;
 using Rubberduck.VBEditor.SafeComWrappers;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
 
 namespace Rubberduck.UI.CodeExplorer.Commands
 {
-    public class RemoveCommand : ExportCommand
+    public class RemoveCommand : CommandBase
```

Comment:
since the parameter must be a `CodeExplorerComponentViewModel`, why is this not a `CodeExplorerCommand`?

Reply:
Changed